### DISCUSSION TITLE
Install on GoPro Silver?

### DISCUSSION THREAD

**Discussion by dimitribeyondgravity** [gopro/labs#434] [Q&A] [answered]
Hi everybody !

GoPro Labs looks awesome, but it only works for Black Editions?
Is there a way to make it work on a silver edition, for example HERO 7 SILVER?

If yes, do you know the reason?

Thanks !

**Comment by dnewman-gpsw** [comment, marked as answer]
It is only for the Black edition cameras.  Main reason, Silver line uses a different SoC (main system chip) so nothing Labs would easily port.  Labs is for GP1 and GP2 powered cameras only: HERO7 Black, HERO8/9/10/11 & MAX.

**Reply by dimitribeyondgravity** [reply]
Ok, thanks for your explanation !
